package.json

```diff
@@ -48,7 +48,7 @@
     "gsn-dock-relay-ganache": "./scripts/gsn-dock-relay-ganache",
     "web": "./restart-relay.sh web",
     "trace": "MODE=trace truffle test",
-    "coverage": "./scripts/run-coverage",
+    "coverage": "yarn generate && ./scripts/run-coverage",
     "clean-coverage": "shx rm -rf coverage coverage.json",
     "generate": "rm -rf ./build/contracts && truffle compile && typechain --target trufflehotfix './build/contracts/**/*.json'",
```

Comment:
any special reason to remove the folder before each build? it slows down repeated build.

Reply:
Yes. If the contract was renamed/removed, it will remain there. The folder should be removed.